wandb/sdk/wandb_run.py

```diff
@@ -103,13 +103,16 @@ class GitSourceDict(TypedDict):
         remote: str
         commit: str
         entrypoint: List[str]
+        args: Sequence[str]
```

Comment:
Why not `List` here? Is this sometimes a tuple? Mixing immutable tuples and mutable lists could introduce sticky bugs.

Reply:
Its declared as a Sequence in the settings object, so I used that